src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -79,8 +81,9 @@ public String getContainer() {
     }
 
     @DataBoundSetter
-    public void setContainer(String container) {
+    public void setContainer(String container) throws IOException {
         this.container = container;
+        checkValue(doCheckContainer(container));
```

Comment:
OK so long as an empty field is just a warning—otherwise you would break saving `/configure` for people not actually using the plugin!

Reply:
I am afraid so, we can allow an empty value, but in that case, there is a not valid configuration that can be saved.
Maybe we can check if the Artifact Manager on S3 it is configured, in that case, you should not save the configuration, but probably it does not work on the first time save.

Is there another way to check the saved values?

Reply:
I tried, it is not possible to know if the Artifact Manager on S3 is enabled when you saved for the first time, so we have to live with the possibility of an empty Bucket name, on the UI there is a warning, so the user is advised.

Reply:
I think this is good enough.